bigtable/google/cloud/bigtable/table.py

```diff
@@ -174,19 +176,35 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
-    def create(self):
+    def create(self, initial_split_keys=None):
         """Creates this table.
 
         .. note::
 
             A create request returns a
             :class:`._generated.table_pb2.Table` but we don't use
             this response.
+
+        :type initial_split_keys: list
+        :param initial_split_keys: The optional list of row keys that will be
+                                    used to initially split the table into
+                                    several tablets.
         """
         table_client = self._instance._client.table_admin_client
         instance_name = self._instance.name
+
+        if initial_split_keys is not None:
+            splits = []
+            for initial_split_key in initial_split_keys:
+                splits.append(
+                    table_admin_messages_v2_pb2.CreateTableRequest.Split(
+                        key=initial_split_key.encode('utf-8')))
```

Comment:
Why are you encoding the key, and not just using the split key supplied by the user?

Reply:
Here, as everywhere, row keys must be bytes.  We surface that to the caller.